### ISSUE TITLE
Code and Dataset Preps

### ISSUE THREAD

**Issue by liellnima** [RolnickLab/snowgalileo#72]
Code:
- [x] New repo
- [x] Docs, Readme, etc.
- [ ] Label 1st version
- [ ] Check with Francis
- [ ] Reproducability / Runnability (tested by Julia)

Later:
- [ ] Clean up Code (comments, remove unsused scripts, ambiguous variable etc)

Dataset:
- [ ] Get data from cluster
- [ ] Talk with Francis on how to document that best
- [ ] Guide on how to reproduce stuff
- [x] Host data somewhere (csv for GoogleEarth, and real datasets for fine-tuning)
- [x] Weights / checkpoints of pretraining + fine-tuning + baseline weights
